cmd/core-data/res/configuration.toml

```diff
@@ -45,7 +45,6 @@ File = './logs/edgex-core-data.log'
   Port = 27017
   Username = ''
   Timeout = 5000
-  Type = 'mongodb'
```

Comment:
Removal kind of concerns me because if I don't have access to the build time logs, I don't have a clear indication of which DB is being used. The way it is now, I can always just go to configuration. And since the Type property has been removed from the DatabaseInfo struct, I can't obtain this information via the sys-mgmt-agent either.

Reply:
I understand your concerns. The removal was IMHO necessary because this option is not read so changing it in the config file would not do anything. 

What we could do is provide a runtime log during DB initialization to see which DB is being launched for each service. The logger call could be added to the newDBClient().

Not sure about how the sys-mgmt-agent works yet but I guess there would be a way to provide this info there as well.